src/Modules/UI/Dnn.PersonaBar.UI/admin/personaBar/scripts/main.js

```diff
@@ -428,16 +428,44 @@ require(['jquery', 'knockout', 'moment', '../util', '../sf', '../config', './../
                     }
                 }
             },
+            /**
+             * in case path is an array, it is expected to be a sorted list of dependent sources.
+             * 
+             * @param path string|array
+             */
             loadBundleScript: function (path) {

```

Comment:
I think that the method deserves a rename...

Reply:
It'd be a breaking change for all modules calling the same.

Reply:
You are right. Oh well, let's leave it